pkg/networkservice/common/heal/client_test.go

```diff
@@ -136,15 +136,13 @@ func TestHealClient_EmptyInit(t *testing.T) {
 		},
 	}
 
-	ctx, cancelFunc := context.WithCancel(context.Background())
+	ctx, cancelFunc := context.WithCancel(token.NewContext(context.Background(), "token", time.Now().Add(time.Hour)))
```

Comment:
Does heal really need to be token aware?

Reply:
This unit test is using updatepath server and currently, to test it locally we should use token in the context

Reply:
Fixed.